include/OpenColorIO/OpenColorIO.h

```diff
@@ -1009,7 +1010,7 @@ class OCIOEXPORT Config
      * * If strict parsing is disabled, return ROLE_DEFAULT (if defined).
      * * If the default role is not defined, return an empty string.
      */
-    OCIO_DEPRECATED("This is now deprecated, please use Config::getColorSpaceFromFilepath().")
+    OCIO_DEPRECATED("This is now deprecated starting at v2.0, please use Config::getColorSpaceFromFilepath().")
```

Comment:
Just a small suggestion. Either wording works fine though.

```suggestion
    OCIO_DEPRECATED("This was marked as deprecated starting in v2.0, please use Config::getColorSpaceFromFilepath().")
```